dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -187,11 +187,11 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL IDI_DLG_BG, IDC_DLG_BG, "STATIC", SS_BITMAP | SS_REALSIZECONTROL, 0, 0, 208, 122
     CONTROL IDI_ROSLOGO_FLAG, IDC_ROSLOGO_WHITE, "STATIC", SS_BITMAP, 176, 1, 32, 26
-    PUSHBUTTON "&Hibernare", IDC_BUTTON_HIBERNATE, 30093, 30049, 22, 20, BS_OWNERDRAW | WS_DISABLED
+    PUSHBUTTON "&Hibernate", IDC_BUTTON_HIBERNATE, 30093, 30049, 22, 20, BS_OWNERDRAW | WS_DISABLED
```

Comment:
Why un-translate it to English?

Reply:
Sorry, I mistakenly did this. I will revert this change.

Reply:
It's fixed now!